patches/media-mojo-services-gpu_mojo_media_client.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/media/mojo/services/gpu_mojo_media_client.cc b/media/mojo/services/gpu_mojo_media_client.cc
+index 6654fa8a5a89218ff6e9b828613a74110a857550..88d614e3a3ebe192b4b81d80334827ccadac5498 100644
+--- a/media/mojo/services/gpu_mojo_media_client.cc
++++ b/media/mojo/services/gpu_mojo_media_client.cc
+@@ -157,7 +157,7 @@ GpuMojoMediaClient::GetSupportedVideoDecoderConfigs() {
+   supported_config_map[VideoDecoderImplementation::kAlternate] =
+       *d3d11_supported_configs_;
+ 
+-#elif BUILDFLAG(USE_CHROMEOS_MEDIA_ACCELERATION)
```

Comment:
why do we need this? Looks like chromeos only?

Reply:
This is an upstream fix from cr82 that hasn't been merged back to cr81. We get a build error without it because we build with vaapi enabled on Linux (which I believe turns on USE_CHROMEOS_MEDIA_ACCELERATION).

Reply:
ok, sounds good